src/logius/fix-md-elements.js

```diff
@@ -0,0 +1,84 @@
+// @ts-check
+// Module logius/fix-table
+// add table class simple to all tables
+
+export const name = "logius/fix-md-elements";
+
+export function run(conf) {
+  addClassTables(conf);
+  addFigureImg(conf);
+  addClassCode(conf);
+}
+// todo check if algorithm is correct!
+function addClassTables(conf) {
+  // todo do nothing if this config is not set?
+  if (!conf.nl_markdownTableClass) {
+    conf.nl_markdownTableClass = "simple";
+  }
+  [...document.querySelectorAll("[data-format=markdown]:not(body)")].forEach(
+    section =>
+      section.querySelectorAll("table").forEach(table => {
+        // table.classList.add("complex");
+        table.classList.add(conf.nl_markdownTableClass);
+      })
+  );
+}
+
+// todo check non happy flows
+function addFigureImg(conf) {
+  if (
+    !conf.nl_markdownEmbedImageInFigure ||
+    conf.nl_markdownEmbedImageInFigure == false
```

Comment:
These are the same, no? 
```suggestion
    !conf.nl_markdownEmbedImageInFigure
```